pkg/reconciler/revision/config/store.go

```diff
@@ -114,9 +129,6 @@ func (s *Store) Load() *Config {
 	if tr, ok := s.UntypedLoad(pkgtracing.ConfigName).(*pkgtracing.Config); ok {
 		cfg.Tracing = tr.DeepCopy()
 	}
-	if as, ok := s.UntypedLoad(asconfig.ConfigName).(*autoscalerconfig.Config); ok {
-		cfg.Autoscaler = as.DeepCopy()
```

Comment:
wondering why we removed the autoscaler config here? Seems unrelated to this PR?

Reply:
it's part of the config in the `apis/config` package.

I needed to pull in the features flag from the above config so it was easier to just embed the struct and drop some duplicate logic in this store